skyagi-web/src/routes/dashboard/+page.svelte

```diff
@@ -1,115 +1,125 @@
 		}
 		return {
 			...characterDataPoint,
 			image: imagePath
-    }})
+		};
+	});
 
-    
-    export const conversations = conversationData
-	let conversationOpenLs = conversations.map((item)=>false)
-	conversationOpenLs[0]=true
+	export const conversations = conversationData;
+	let conversationOpenLs = conversations.map(item => false);
+	conversationOpenLs[0] = true;
 
 	function handleResumeRoomClick() {
-		window.location.href = '/room/resume-room/' + activeId
+		window.location.href = '/room/resume-room/' + activeId;
 	}
 
-    function handleCreateAgentClick() {
-        window.location.href = '/agent/create'
-    }
+	function handleCreateAgentClick() {
+		window.location.href = '/agent/create';
+	}
 
-    function handleCreateRoomClick() {
-        window.location.href = '/room/new'
-    }
+	function handleCreateRoomClick() {
+		window.location.href = '/room/new';
+	}
 </script>
 
 <div id="globalGrid">
+	<div>
+		<div class="conversationscroller">
+			<Accordion
+				id="conversationBoard"
+				activeClasses="bg-gray-800 text-white focus:ring-4 focus:ring-blue-800 text-2xl"
+				inactiveClasses="text-gray-400 hover:bg-gray-800 text-2xl"
+			>
+				{#each conversations as conversation, i}
+					<Conversation
+						conversationIndex={i + 1}
+						conversationSummary={conversation}
+						conversationId={conversation.conversationId}
+						bind:activeId
+					/>
+				{/each}
+			</Accordion>
+		</div>
+		<div id="buttonGrid">
+			<Button size="xl" on:click={handleResumeRoomClick}>Resume to the selected conversation</Button
+			>
+			<Button size="xl" on:click={handleCreateRoomClick}>Create new conversation</Button>
+			<Button size="xl" on:click={handleCreateAgentClick}>Create new agent</Button>
+		</div>
+	</div>
 
-    <div>
-        <Accordion id="conversationBoard">
-            {#each conversations as conversation, i}
-                <Conversation conversationIndex={i+1} 
-				conversationSummary = {conversation} 
-				conversationId = {conversation.conversationId}
-				bind:activeId={activeId}>
-                </Conversation>
-				
-            {/each}
-        </Accordion>
-        <div id="buttonGrid">
-			<Button on:click={handleResumeRoomClick}>
-				Resume to the selected conversation
-			</Button>
-            <Button on:click={handleCreateRoomClick}>
-                Create new conversation
-            </Button>
-            <Button on:click={handleCreateAgentClick}>
-                Create new agent
-            </Button>
-        </div>
-    </div>
-    
-    <div class="scroller">
-        {#each characters as character, i}
-            <a href= "agent/{character.id}">
-            <div class="characterInfoSet">
-                <Character {character} imageUrl={character.image}>
-                </Character>
-            </div>
-            </a>
-        {/each}
-    </div>
+	<div class="scroller">
+		{#each characters as character, i}
+			<a href="agent/{character.id}">
```

Comment:
The focus area of the avatar seems too larger than the size of the avatar. 
Like if click the area in the middle of C and A-Serena,  the agent page will be opened.
Tagging @serenaqiaoo myself here so I will fix this in another PR.
<img width="622" alt="Screenshot 2023-06-24 at 1 20 00 AM" src="https://github.com/litanlitudan/skyagi/assets/121361687/a60658d7-27a1-4a1b-906b-1565111c543d">